build.gradle

```diff
@@ -30,15 +30,14 @@ plugins {
     id 'org.liquibase.gradle' version '2.0.1'
     id "org.gradle.test-retry" version "1.2.0"
     id 'antlr'
-    id 'com.github.spotbugs' version '4.7.0'
+    id 'com.github.spotbugs' version '4.7.1'
```

Comment:
For others taking a look at this PR, the `build.gradle` files are the only two non-java files that are changed. All the other changes are due to spotless suggestions. The bulk of the changes are whitespace and documentation formatting.